sites/fast-website/src/app/components/card-section/card-section.styles.ts

```diff
@@ -1,12 +1,13 @@
 import { css } from "@microsoft/fast-element";
 import { display } from "@microsoft/fast-foundation";
+import { siteBreakpoints } from "../../style/constants";
 
 export const CardSectionStyles = css`
     :host {
         --flow: column;
     }
 
-    @media screen and (max-width: 1000px) {
+    @media screen and (max-width: ${siteBreakpoints[2]}) {
```

Comment:
I understand the thinking behind having the breakpoints in an array and accessing it. However, I think the readability suffers a bit. I wonder if there's an alternative way to represent them that creates greater clarity at the point where they are used. Not a huge deal, but something worth thinking about.

Reply:
We followed this pattern before in react with `BreakpointTracker`. I'm not saying it's the right way but I suggested doing it this way to follow that pattern we have used in the past.

Reply:
Reverting this to fixed numbers for now.  See Chris' comment below.